administrator/components/com_content/src/Model/ArticleModel.php

```diff
@@ -784,6 +784,7 @@ public function save($data)
 		if ($input->get('task') == 'save2copy')
 		{
 			$origTable = $this->getTable();
+			$data['state'] = 0; 
```

Comment:
```suggestion
			$data['state'] = 0;
```
Please remove the space here at the end of the line to fix the code style test failing in drone (our tool for automated checks).